lib/cb/clients/category.rb

```diff
@@ -25,15 +24,14 @@ def self.search
             end
           end
 
-          my_api.append_api_responses(categoryList, json_hash['ResponseCategories'])
+          cb_client.append_api_responses(categoryList, json_hash['ResponseCategories'])
```

Comment:
We could remove 2 lines (eh, only 2 lines) from the above code using `fetch`:

```ruby
if json_hash.key?('ResponseCategories')
  categoryList = []
  json_hash.fetch('ResponseCategories', {}).fetch('Categories', {}).fetch('Category', []).each do |cat|
    categoryList << Models::Category.new(cat)
  end

  cb_client.append_api_responses(categoryList, json_hash['ResponseCategories'])
end
```

Code climate is flagging this guy as a "complex method".